packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -169,6 +169,11 @@ export function createAutocompleteDom<TItem extends BaseItem>({
       type: 'button',
       class: classNames.detachedCancelButton,
       textContent: translations.detachedCancelButtonText,
+      // Prevent environment `onTouchStart` behavior from closing the panel
+      // if it's initiated from this button
+      onTouchStart(event: TouchEvent) {
+        event.stopPropagation();
+      },
       onClick(event: MouseEvent) {
         event.stopPropagation();
```

Comment:
This was introduced in the initial PR, does it need to stay?

Reply:
Nice point, I don't see any value for it now that we have correctly targeted the origin of the issue. I'll remove it.